src/python/friendzone/utils/unwrap_inputs.py

```diff
@@ -13,11 +13,45 @@
 # limitations under the License.
 
 from simde import TotalEnergy, EnergyNuclearGradientStdVectorD
+import numpy as np
 
+import numpy as np
```

Comment:
```suggestion
from simde import TotalEnergy, EnergyNuclearGradientStdVectorD
import numpy as np
```

Numpy is imported twice, it should only be imported once.